invoice_double_validation/models/two_validations_invoice.py

```diff
@@ -0,0 +1,45 @@
+# -*- encoding: utf-8 -*-
+############################################################################
+#    Module Writen For Odoo, Open Source Management Solution
+#
+#    Copyright (c) 2011 Vauxoo - http://www.vauxoo.com
+#    All Rights Reserved.
+#    info Vauxoo (info@vauxoo.com)
+#    coded by: hugo@vauxoo.com
+#    planned by: Nhomar Hernandez <nhomar@vauxoo.com>
+############################################################################
+
+from openerp.osv import fields, osv
+from openerp.tools.translate import _
+
+
+class AccountInvoice(osv.Model):
+    _inherit = 'account.invoice'
+
+    def get_states(self, cr, uid, context):
+        return [
+            ('draft', 'Draft'),
+            ('proforma', 'Pro-forma'),
+            ('proforma2', 'Pro-forma'),
+            ('validate', _('By validating')),
+            ('open', 'Open'),
+            ('paid', 'Paid'),
+            ('cancel', 'Cancelled'), ]
+
+    _columns = {
+        'state': fields.selection(
```

Comment:
@hugho-ad
Can you change the field `state` with new api and use `selection_add` param to add a new value in invoice?
http://odoo-new-api-guide-line.readthedocs.org/en/latest/fields.html#selection